src/Tag/StreamTag.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the PhpM3u8 package.
+ *
+ * (c) Chrisyue <http://chrisyue.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Chrisyue\PhpM3u8\Tag;
+
+class StreamTag extends AbstractTag
+{
+    use SingleValueTagTrait;
```

Comment:
why not use `AttributesValueTagTrait`

https://github.com/chrisyue/php-m3u8/blob/develop/src/Tag/AttributesValueTagTrait.php

you could check `KeyTag` class as an example.